application.go

```diff
@@ -217,17 +209,20 @@ func (a *Application) Suspend(f func()) bool {
 	var err error
 	a.screen, err = tcell.NewScreen()
 	if err != nil {
+		a.suspendMutex.Unlock()
```

Comment:
Instead of putting `Unlock()`'s everywhere, a `defer Unlock()` at the top is probably better.